package.json

```diff
@@ -21,6 +21,7 @@
     "@monaco-editor/react": "^4.6.0",
     "@radix-ui/react-icons": "^1.3.1",
     "@xyflow/react": "^12.3.0",
+    "html-to-image": "^1.11.11",
```

Comment:
if you use html to image, then you probably should remove html to canvas

Reply:
html2canvas removed